x-pack/plugins/ml/index.js

```diff
@@ -81,7 +81,7 @@ export const ml = (kibana) => {
         const config = server.config();
         return {
           kbnIndex: config.get('kibana.index'),
-          esServerUrl: config.get('elasticsearch.url'),
+          esServerUrl: config.get('elasticsearch.hosts')[0]
```

Comment:
@elastic/ml-ui may want to improve what they're doing here now that we can get a full array.

Reply:
I just pushed a commit that removes this, 070ecd6.  I couldn't find any references to it in and checked commit history 2 years back looking for when it was implemented but didn't have any luck.
@elastic/ml-ui can you verify this is okay?

Reply:
@jbudz yes, it hasn't been used since Prelert joined elastic!
We used to query elasticsearch directly and not use the elasticsearch-js client.
The same goes for `kbnIndex`, but we can remove that one ourselves.

Reply:
awesome thanks @jgowdyelastic!